hash/apple2_flop_misc.xml

```diff
@@ -721,6 +721,18 @@ Generated by SLIST 0.2.1
 				<rom name="alien (1982)(avalon hill).do" size="143360" crc="8928926a" sha1="cf8861b433def03864b4611d21be058d86fb53da" />
 			</dataarea>
 		</part>
+	</software>	
+
+	<software name="alephbai">
+		<description>Aleph Baiters</description>
+		<year>1983</year>
+		<publisher>Gesher Educational Affiliates</publisher>
+
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="143360">
+				<rom name="1983-aleph-baiters.dsk" size="143360" crc="93020d00" sha1="8a246f2a57f97f9119b01834d48f70a266308f0d" offset="0"/>
```

Comment:
Also could you remove all of the ``offset="0"`` attributes? They aren't strictly needed since 0 is the default (and generally only make sense for ROMs) and aren't used anywhere else in this software list.